glusterd2/commands/volumes/volume-profile-utils.go

```diff
@@ -0,0 +1,99 @@
+package volumecommands
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/volume"
+)
+
+// Fops
+var fops = []string{
+	"NULL",
+	"STAT",
+	"READLINK",
+	"MKNOD",
+	"MKDIR",
+	"UNLINK",
+	"RMDIR",
+	"SYMLINK",
+	"RENAME",
+	"LINK",
+	"TRUNCATE",
+	"OPEN",
+	"READ",
+	"WRITE",
+	"STATFS",
+	"FLUSH",
+	"FSYNC", /* 16 */
+	"SETXATTR",
+	"GETXATTR",
+	"REMOVEXATTR",
+	"OPENDIR",
+	"FSYNCDIR",
+	"ACCESS",
+	"CREATE",
+	"FTRUNCATE",
+	"FSTAT", /* 25 */
+	"LK",
+	"LOOKUP",
+	"READDIR",
+	"INODELK",
+	"FINODELK",
+	"ENTRYLK",
+	"FENTRYLK",
+	"XATTROP",
+	"FXATTROP",
+	"FGETXATTR",
+	"FSETXATTR",
+	"RCHECKSUM",
+	"SETATTR",
+	"FSETATTR",
+	"READDIRP",
+	"FORGET",
+	"RELEASE",
+	"RELEASEDIR",
+	"GETSPEC",
+	"FREMOVEXATTR",
+	"FALLOCATE",
+	"DISCARD",
+	"ZEROFILL",
+	"IPC",
+	"SEEK",
+	"LEASE",
+	"COMPOUND",
+	"GETACTIVELK",
+	"SETACTIVELK",
+	"PUT",
+	"ICREATE",
+	"NAMELINK",
+	"MAXVALUE",
+}
+
+var profileSessionKeys = [...]string{"io-stats.count-fop-hits", "io-stats.latency-measurement"}
```

Comment:
IIUC, these options are applied while taking profiles? Are we turning these off when profile is switched off? I couldn't figure out that code snippet?